public/scripts/utils.js

```diff
@@ -33,6 +34,74 @@ export function isValidUrl(value) {
     }
 }
 
+/**
+ * Converts string to a value of a given type. Includes pythonista-friendly aliases.
+ * @param {string|SlashCommandClosure} value String value
+ * @param {string} type Type to convert to
+ * @returns {any} Converted value
+ */
+export function convertValueType(value, type) {
+    if (value instanceof SlashCommandClosure || typeof type !== 'string') {
+        return value;
+    }
+
+    type = type.trim().toLowerCase();
+
+    switch (type) {
+        case 'string':
+        case 'str':
+            return String(value);
+
+        case 'null':
+            return null;
+
+        case 'undefined':
+        case 'none':
+            return undefined;
```

Comment:
Setting the property to undefined will just remove it, as the object is serialized right away when the command makes its return.
Doesn't hurt though, another option to *unset* a property inside an object.